src/app.js

```
@@ -43,18 +44,50 @@
   }
 
   init() {
+    this.renderActivityCountdown();
     this.renderProducts();
     this.updateCartCount();
   }
 
+  renderActivityCountdown() {
+    const header = document.querySelector('header');
```

ESLint

[error] Extra semicolon.

(semi)